Harmony/Internal/MethodPatcher.cs

```diff
@@ -571,7 +586,6 @@ void EmitCallParameter(MethodInfo patch, Dictionary<string, LocalBuilder> variab
 				// treat __result var special
 				if (patchParam.Name == RESULT_VAR)
 				{
-					var returnType = AccessTools.GetReturnedType(original);
```

Comment:
Weird. For me this looks like `returnType` isn't assigned anymore but used in the next line.

Reply:
There's a field with same value in the class